benchmarks/benchmarks.rb

```diff
@@ -70,6 +70,26 @@ def keyword_args(a:, b:)
         100
       end
       #{benchmark_gem.memoization_method} :keyword_args
+
+      def positional_and_keyword_args(a, b:)
+        100
```

Comment:
Do we have an issue to fill these in?

Reply:
What would we fill these in with? I don't think the result would impact the benchmark since we're only benchmarking cache value returns, right?

Reply:
Ah yup you're totally right, PR lgtm. Can merge this if you approve!

Also, looks like I lost permission to push to this repo. Any chance you could check on that?

Reply:
Whoops, sorry! Should be fixed now!